src/main/java/de/zalando/aruha/nakadi/controller/ExceptionHandling.java

```diff
@@ -36,5 +38,16 @@ private String generateErrorTraceId() {
         return "ETI" + RandomStringUtils.randomAlphanumeric(24);
     }
 
-
+    @Override
+    @ExceptionHandler
+    public ResponseEntity<Problem> handleMessageNotReadableException(final HttpMessageNotReadableException exception, final NativeWebRequest request) {
```

Comment:
It's not very clear why this code exists. It handles `JsonMappingException` in a very specific way. Maybe we should add a test for that or just document this code, explaining the motivation.